pages/linux/lxi.md

```diff
@@ -0,0 +1,24 @@
+# lxi
+
+> Tool for controling LXI compatible instruments such as oscilloscopes.
+> More information: <https://github.com/lxi-tools/lxi-tools>.
+
+- Discover LXI devices on available networks:
```

Comment:
```suggestion
- Discover `LXI` devices on available networks:
```

Reply:
Detto about the usage of \` - is there a justification for such change, please? (to know better in the future)

Reply:
See previous review for my reply

Reply:
No, I'm not sure backticks make sense here either.

Reply:
I had a feeling they would be controversial. Feel free to resolve if agreed however leave open for now